design-toolkit/components/src/components/tree/index.tsx

```diff
@@ -102,6 +102,17 @@ const TreeLines = memo(function TreeLines({
   });
 });
 
+/**
+ * Tree - Hierarchical tree view with optional drag-and-drop and visibility
+ *
+ * Renders a selectable tree with support for nested items, drag-and-drop,
+ * selection modes, and visibility controls. Use Tree.Item to define nodes.
+ *
+ * @example
+ * <Tree>
+ *   <Tree.Item>Node</Tree.Item>
```

Comment:
Can you flush this one out a little bit more too?

Reply:
I've added the first two examples from *.mdx.